frontend/src/components/AdminModals/AdminSuspendModal.js

```diff
@@ -27,22 +27,25 @@ function AdminSuspendModal(props) {
     </button>
   );
 
+  const toggleSuspendText = (isSuspended) => {
```

Comment:
I'd probably just call this `modalText` or something, since it just returns text rather than toggling anything.